src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/VMDetailsSection.scss

```diff
@@ -0,0 +1,3 @@
+.instancetypes-vm-details-section {
+  margin-top: var(--pf-global--spacer--lg);
+}
```

Comment:
I can see "No newline at end of file".
So maybe adding a new line to this file would be better :)

Reply:
Done!